hadoop-common-project/hadoop-common/src/main/resources/core-default.xml

```diff
@@ -2404,6 +2404,16 @@ The switch to turn S3A auditing on or off.
 
 
 <!-- ipc properties -->
+<property>
+  <name>ipc.client.async.calls.check.enable</name>
```

Comment:
Instead of adding another conf, how about having ipc.client.async.calls.max set to -1 for disabling it?

Reply:
Hi, @szetszwo . Thanks a lot for reviewing. Looks Good because it can prevent introduce a new configuraton.
We can modify code like this? Let's here what other people say and discuss it.

```java
  public boolean isAsyncCallCheckEabled() {
    return maxAsyncCalls >= 0;
  }
```

Hi, @KeeProMise . Please cc. this, thanks a lot.

Reply:
Hi @hfutatzhanghb It's a good idea.